README.md

```diff
@@ -32,4 +32,4 @@ Run npm install
 * Ed - 
 * Matt - scribe :(
 * Arun
-* Kevin V.
+* Kevin Vensel
```

Comment:
Can you resolve conflicts before putting this up for review? I've added a newer readme, so please do a git pull or merge, fix merge issues, and then recommit.